exonum/src/blockchain/mod.rs

```diff
@@ -129,7 +128,7 @@ impl Blockchain {
     /// Returns the `VecMap` for all services. This is a map which
     /// contains service identifiers and service interfaces. The VecMap
     /// allows proceeding from the service identifier to the service itself.
-    pub fn service_map(&self) -> &Arc<VecMap<Box<dyn Service>>> {
+    pub fn service_map(&self) -> &Arc<HashMap<u16, Box<dyn Service>>> {
```

Comment:
Documentation comment should be updated accordingly as well. I guess we can avoid specifying concrete type of the map.